source/vir/src/sst_vars.rs

```diff
@@ -4,16 +4,27 @@ use crate::sst::{Stm, StmX, UniqueIdent};
 use std::collections::{HashMap, HashSet};
 use std::sync::Arc;
 
+fn to_ident_set(input: &HashSet<UniqueIdent>) -> HashSet<Arc<String>> {
+    let mut output = HashSet::new();
+    for item in input {
+        output.insert(item.0.clone());
+    }
+    output
+}
+
 // Compute:
 // - which variables have definitely been assigned to up to each statement
 // - which variables have been modified within each statement
+pub type AssingMap = HashMap<*const Spanned<StmX>, HashSet<Arc<String>>>;
```

Comment:
This should probably be `AssignMap`, right?

Reply:
Yep, you are right.